### DISCUSSION TITLE
Ability to adjust reverb, chorus and other midi controllers

### DISCUSSION THREAD

**Discussion by wiwikuan** [alda-lang/alda#452] [Ideas]
# 🔧 Feature request 🔧

## Description

Alda is great! But the Java Gervill MIDI player's default reverb setting is just too high (I don't want that much reverb in my music), and I don't have control to any other standard MIDI CC message.

## Examples

It would be nice if I can say `(reverb! 0)` to turn off the reverb globally. And it would be even nicer if I can say something like `(chrous 40)` or `(CC1 100)` to set any other MIDI controllers. 

Also, the value of MIDI controllers are typically 0-127 (instead of 0-100), I don't know if it's better to map the values from 0-100 to 0-127, or just use 0-127 in the code.

**Comment by daveyarwood** [comment]
I love this idea! I just played around with this a bit locally and I think it could be a lot of fun.

I especially like the idea of providing a general-purpose `control-change` function so that if Alda doesn't have a dedicated function like `reverb` for the CC message you have in mind, you can still use it by specifying the CC number.

Looking at [this list](https://www.midi.org/specifications-old/item/table-3-control-change-messages-data-bytes-2), I think it would also make sense to add dedicated functions for a handful of common ones like reverb, chorus, and vibrato. Maybe I'll start with those 3, but if anyone has suggestions for others that we could add, I would be open to it.

**Comment by wiwikuan** [comment]
I think Mod wheel (CC#1), Reverb (CC#91), chorus (CC#93) and sustain pedal (CC#64) are the most essential ones.
